### ISSUE TITLE
Expiry month and year with two select fields

### ISSUE THREAD

**Issue by Cloudchoice** [jessepollak/card#340]
Hello, is there any solution to the input SELECT problem not being recognized? 

I would like to use two selects one for month and one for year. Apparently the page "witcherstore.com" solve the problem.

**Comment by kambanwait** [comment]
Not sure what problem you're having with this; I've not seen any issues when implementing it 🤔

**Comment by Cloudchoice** [comment]
Everything works  smoothly **except the select**. I am using this example code:

 formSelectors: {
        numberInput: 'input[name="CARD"]',
        nameInput: 'input[name="CARD_NAME"]',
        expiryInput: 'select[name="CARD_2"], select[name="CARD_3"]',
        cvcInput: 'input[name="CARD_CODE"]'
    },


<select name="CARD_2" id="CARD_2">
<option value="">Month</option>
<option value="1">01</option>
<option value="2">02</option>
<option value="3">03</option>
<option value="4">04</option>
<option value="5">05</option>
<option value="6">06</option>
<option value="7">07</option>
<option value="8">08</option>
<option value="9">09</option>
<option value="10">10</option>
<option value="11">11</option>
<option value="12">12</option>
</select>

<select name="CARD_3" id="CARD_3">
<option  value="">Year</option>
<option value="2016">16</option>
<option value="2017">17</option>
<option value="2018">18</option>
<option value="2019">19</option>
<option value="2020">20</option>
<option value="2021">21</option>
<option value="2022">22</option>
<option value="2023">23</option>
<option value="2024">24</option>
<option value="2025">25</option>
<option value="2025">26</option>
</select>

**Comment by robsonsobral** [comment]
Same problem here. It works with one or two `inputs`, but not with `select`.

**Comment by Cloudchoice** [comment]
YES, the site "witcherstore.com" solved the problem and was able to use select in the month / year payment field, but I did not figure out how.

**Comment by nancorrea** [comment]
Hey guys, solve that out

`year_selector = 'select[name="data[Source][exp_year]"]';
		month_selector = 'select[name="data[Source][exp_month]"]';
		
		$(month_selector).change(function(){
			year = $(year_selector).val();
			year = year == '' ? '**': year;
			$('.jp-card-expiry').text($(this).val()+'/'+year);
		});
		$(year_selector).change(function(){
			month = $(month_selector).val();
			month = month == '' ? '**': month;
			$('.jp-card-expiry').text(month+'/'+$(this).val());
		});
		$(month_selector).add(year_selector)
			.on('focus', function(){
				$('.jp-card-expiry').addClass('jp-card-focused');
			})
			.on('blur', function(){
				$('.jp-card-expiry').removeClass('jp-card-focused');
			});`

**Comment by jbarnicle** [comment]
@nancorrea - I think you're missing the the () to invoke your IIFE,  or am I missing something?

**Comment by topsub** [comment]
This worked for me. Sucks this has to be a work around

**Comment by atakanaydin** [comment]
Thats it. Thanx

**Comment by CliqLabs** [comment]
Bump.  I'm having the same issue here.  The vast majority of credit card expiration fields out there use select elements.  As such, the developer should update Card to allow for them.
